includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -100,19 +100,17 @@ function perflab_aao_autoloaded_options_test() {
  * @return int autoloaded data in bytes.
  */
 function perflab_aao_autoloaded_options_size() {
-	global $wpdb;
-
-	$autoload_values = perflab_aao_get_autoload_values_to_autoload();
-
-	return (int) $wpdb->get_var(
-		$wpdb->prepare(
-			sprintf(
-				"SELECT SUM(LENGTH(option_value)) FROM $wpdb->options WHERE autoload IN (%s)",
-				implode( ',', array_fill( 0, count( $autoload_values ), '%s' ) )
-			),
-			$autoload_values
-		)
-	);
+	wp_load_alloptions();
+
+	$all_options = wp_cache_get( 'alloptions', 'options' );
```

Comment:
Why not take the return value directly from `wp_load_alloptions();` here?

Reply:
thanks, commit done